paasta_tools/paasta_cli/cmds/rollback.py

```diff
@@ -37,9 +38,9 @@ def add_subparser(subparsers):
                              required=True,
                              )
     list_parser.add_argument('-i', '--instance',
```

Comment:
can you make this "instances"? In python you can abbreviate it too, so you can just say --instance and it will still work.